test/src/androidTest/kotlin/io/realm/shared/EncryptionTests.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.shared
+
+import io.realm.Realm
+import io.realm.RealmConfiguration
+import io.realm.util.PlatformUtils
+import test.Sample
+import kotlin.random.Random
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertFails
+
+class EncryptionTests {
```

Comment:
Maybe put some doc explaining what this class exactly tests compared to RealmConfigurationTests?